doc/source/gettingstarted.rst

```diff
@@ -54,36 +54,32 @@ Now download the file (example below) and save it as values.yaml file
 ::
     araali:
         workload_id: <wrk-id-variable>
-        cluster_name: bar
-        fog: foo
-        zone: <name-you-specified>
-        app: k8s-nodes
-        enforce: true
-        upgrade: true
-        autok8s_image: quay.io/araalinetworks/autok8s:prod
-        fw_image: quay.io/araalinetworks/araali_fw:prod
-        fw_init_image: quay.io/araalinetworks/araali_fw_init:prod
+        zone: <zone>
 
 
 Step 3A: Installation for Kubernetes
 ***********************************
 **Add Araali Repo and Install the Helm Chart.**
 
-1. Add Repo::
+1. Add Repo*::
 
     helm repo add araali-helm https://araalinetworks.github.io/araali-helm/
 
+    #If you have already added the repo, instead run this command periodically to keep you repo up to date:
+
+    helm repo update araali-helm https://araalinetworks.github.io/araali-helm/
```

Comment:
Just `helm repo update` delete the rest.

Reply:
done!